src/theme/base/_variables.scss

```diff
@@ -3,13 +3,41 @@ $font-main: 'Open Sans', sans-serif;
 $font-fjalla-one: 'Fjalla One', sans-serif;
 
 // Palette
+$primary100: #179A6F;
+$primary200: #24A87C;
+$primary300: #32B88C;
+$primary400: #3DC598;
+$primary500: #69D5B1;
+$primary600: #76DFBC;
+$primary700: #8EEBCC;
+$primary800: #B0F3DD;
+$primary900: #C1FDE9;
+
+$secondary100: #76AC53;
+$secondary200: #85B863;
+$secondary300: #97CC74;
+$secondary400: #A7DC84;
+$secondary500: #B6EA94;
+$secondary600: #C5F4A7;
+$secondary700: #D2F8B9;
+$secondary800: #DDFBCA;
+$secondary900: #E9FADD;
+
+$tertiary100: #AE382F;
+$tertiary200: #C5443A;
+$tertiary300: #D34F44;
+$tertiary400: #E35F54;
+$tertiary500: #EB6E64;
+$tertiary600: #EA7A71;
+$tertiary700: #EE867E;
+$tertiary800: #F29891;
+$tertiary900: #F6AFA9;
+
 $color-main: #EB6E64;
```

Comment:
We have two variables for same color `#EB6E64` - `$tertiary500` & `$color-main`.
We should use just one variable for it.

Reply:
Good catch! I've removed `$color-main`. Thanks!